frontend/src/components/Common/ReportDialog/index.tsx

```diff
@@ -107,6 +118,32 @@ const ReportDialog = memo(
       theme,
     ]);
 
+    const renderedPdfLoadingProgressBar = useMemo(() => {
+      if (!documentIsLoading) {
+        return null;
+      }
+      return (
+        <Box className={classes.documentLoadingContainer}>
+          <Typography
+            className={classes.documentLoaderText}
+            variant="body1"
+            component="span"
+          >
+            {t('Loading document')}
+          </Typography>
+          <span className={classes.documentLoaderDot}>.</span>
+          <span className={classes.documentLoaderDot}>.</span>
+          <span className={classes.documentLoaderDot}>.</span>
```

Comment:
Let's wrap this in a small reusable component with a self-explanatory name please?

Reply:
I meant move the blinkingDots into their own component ;) @HarryMytilinaios

Reply:
Updated

Reply:
Am I missing a commit maybe?

Reply:
Ohhh I understood to move the whole component... Sorry for the misunderstanding chaning it